docs/ReleaseProcess.md

```diff
@@ -99,37 +130,6 @@ Servicing a release has two main scenarios:
-   >**IMPORTANT:** While awaiting sign-off, **only bug-fix PRs are allowed into the `release/X.Y` branch**. In case bug-fix PRs are merged into the `release/X.Y` branch, it is crucial to **designate the latest commit `release/X.Y` branch as the commit for release**.
-1. Once the `sign-off` label has been added, on the `microsoft/ebpf-for-windows` repo's `release/X.Y` branch, create a tag for the *latest commit on the `release/X.Y` branch*. The tag should reflect the version number being released and adhere to the following notation: "`Release-vX.Y.Z`".
-1. The tag creation will automatically trigger the "`CI/CD - Release validation`" workflow for the `release/X.Y` branch: wait for it to complete successfully.
-    >**NOTE:** If the release validation fails, it is the responsibility of the release manager to trigger further investigations, including eventually the submission of necessary issues. Once the issue(s) is(are) resolved, potentially through other PRs, it is important to **delete the previous tag**. Subsequently, recreate the same tag for the *latest commit* on the `release/X.Y` branch, wait for the "`CI/CD - Release validation`" workflow to complete successfully.
-1. Go to the repo on GitHub and click on "`<Code>`" and click on right the "`Create a new release`" link.
-1. Click on the "`Choose a tag`" combo box and select the tag with new "`Release vX.Y.Z`" version number, as created earlier.
-1. Fill in the release title as "`vX.Y.Z`" (replace "`X.Y.Z`" with the version number being released).
-1. Manually enter release notes or click "`Generate release notes`" and then edit as desired.
-1. Attach the `.msi`, the (non-redist) `.nupkg`, the `Build-x64-[Release|Debug].zip` and the `Build-x64-native-only-[Release|Debug].X.Y.Z.zip` build (from the CI/CD artifacts), by dropping them in the "`Attach binaries by dropping them here or selecting them.`" area. For example, the file list for `v0.11.0` should be:
-
-    - *Build-x64-native-only-Debug.0.11.0.zip*
-    - *Build-x64-native-only-Release.0.11.0.zip*
-    - *ebpf-for-windows.0.11.0.msi*
-    - *eBPF-for-Windows.0.11.0.nupkg*
-    - *Build-x64-Debug.zip*
-    - *Build-x64-Release.zip*
-
-1. Check the "`Set as a pre-release`" checkbox, unless the release is production-signed.
-1. Once the uploads are complete, click "`Publish release`".
-
-### Publishing the Release to NuGet.org
-
-Upload the (**non-redist**) `.nupkg` file to [NuGet.org](https://www.nuget.org/) (the metadata inside the `.nuget` package itself will automatically populate all the other form fields).
+1. Mark the draft PR as "Ready for review".
+3. Submit the PR for review (from its draft state), and wait for it to be approved and merged into the main repo's "`main`" branch.
+4. Create a tag for the PR's commit number, on the main repo's "`main`" branch, with meaningful name (i.e., "*RI-from-release-vX.Y.Z*").
```

Comment:
Why are you tagging `main`? Tags are generally reserved for releases only.

Reply:
@gtrevi - is it needed to tag the main if we ever RI from a release branch?

Reply:
Removed the line about tagging.